include/rocksdb/utilities/backupable_db.h

```diff
@@ -142,6 +142,20 @@ struct BackupableDBOptions {
   }
 };
 
+struct CreateBackupOptions {
+  // Flush will always trigger if 2PC is enabled.
+  // If write-ahead logs are disabled, set flush_before_backup=true to
+  // avoid losing unflushed key/value pairs from the memtable.
+  bool flush_before_backup = false;
+
+  // Callback for reporting progress.
+  std::function<void()> progress_callback = []() {};
+
+  // If false, background_thread_cpu_priority is ignored.
+  bool enable_update_background_thread_cpu_priority = false;
+  port::CpuPriority background_thread_cpu_priority = port::CpuPriority::kNormal;
```

Comment:
CpuPriority  doesn't have to belong to port as it is platform independent. I think we can place it simply in options.h.

Reply:
I thought the `port` module needs to be independent of rocks' public include files. But I saw `port_win.h` includes `options.h`, so I'll `CpuPriority` to `options.h`